### DISCUSSION TITLE
v0.4.8 Pre-release Thread

### DISCUSSION THREAD

**Discussion by masenf** [reflex-dev/reflex#3079] [Release]
# Try out the Prerelease

Please report any problems you encounter in this discussion thread or as separate issues on reflex-dev/reflex.

```console
pip install 'reflex~=0.4.8a'
```

# Schedule

2024-04-11 18:15 PDT: 0.4.8a1 is released for general testing
2024-04-15 15:13 PDT: General release of 0.4.8

# Release Notes

### new `transpile_packages` field in rx.Component

For packages on npm which are not exported as modules, these can now be transpiled by Next.JS and used directly.

* [REF-2392] Expose next.config.js transpilePackages key by @masenf in https://github.com/reflex-dev/reflex/pull/3006

### Use `bun` as a package manager on windows

_Much faster_ package installation on windows!

x64 only (no x86 or ARM support)

* [REF-1586] Use bun as a package manager on windows by @masenf in https://github.com/reflex-dev/reflex/pull/2359

### Upload with no files selected still calls event handler

Note: you need to specify a default value for your files parameter to take advantage of this change.

* Fix For When user selects Empty files using File Upload by @Yummy-Yums in https://github.com/reflex-dev/reflex/pull/3051

### Enum types are now serialized to their value

* added serializer for enums by @wassafshahzad in https://github.com/reflex-dev/reflex/pull/3058

### Automatic tuple unpacking also works for Component children

* [REF-2524]Tuple unpacking should apply at component level by @ElijahAhianyo in https://github.com/reflex-dev/reflex/pull/3062

## Bug Fixes

* minimal fix for upload_url in rx.download by @Lendemor in https://github.com/reflex-dev/reflex/pull/3073
* Force pydantic v1 for sqlmodel compatibility by @masenf in https://github.com/reflex-dev/reflex/pull/3026
* Fix annotated EventHandler by @masenf in https://github.com/reflex-dev/reflex/pull/3076

## Experimental

* add run_in_thread by @Lendemor in https://github.com/reflex-dev/reflex/pull/3077

## Other Changes

* chore: fix some typos in comments by @hugehope in https://github.com/reflex-dev/reflex/pull/3071
* fix: removed .git in template URL by @wassafshahzad in https://github.com/reflex-dev/reflex/pull/3049

## New Contributors
* @hugehope made their first contribution in https://github.com/reflex-dev/reflex/pull/3071

**Full Changelog**: https://github.com/reflex-dev/reflex/compare/v0.4.7...reflex-0.4.8

**Comment by yellowbean** [comment]
A minor tips , in case some os throw error "invalid requirement" you can try with 

`pip install -U reflex --pre`

**Comment by masenf** [comment]
0.4.8 is released
